SpellChecker.java

```diff
@@ -11,25 +11,65 @@ public static void main(String[] args) {
 	}
 
 	public static String tail(String str) {
-		// Your code goes here
+		return str.substring(1);
 	}
 
 	public static int levenshtein(String word1, String word2) {
-		// Your code goes here
+		word1 = word1.toLowerCase();
+        word2 = word2.toLowerCase();
+
+		int word1Length = word1.length();
+		int word2Length = word2.length();
+
+		if (word1Length == 0 && word2Length == 0) {
+			return 0;
+		}
+		if (word1Length == 0) {
+			return word2Length;
+		}
+		if (word2Length == 0) {
+			return word1Length;
+		}
```

Comment:
Your first condition covered by next validations - it's unnecessary here
```suggestion
		if (word1Length == 0) {
			return word2Length;
		}
		if (word2Length == 0) {
			return word1Length;
		}
```